vermouth/processors/annotate_idrs.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2024 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Provides processors that can add and remove IDR specific bonds
+"""
+
+import functools
+
+from .processor import Processor
+from ..graph_utils import make_residue_graph
+from ..rcsu.go_utils import _in_resid_region
+from ..log_helpers import StyleAdapter, get_logger
+LOGGER = StyleAdapter(get_logger(__name__))
+
+
+def annotate_disorder(molecule, idr_regions):
+    """
+    annotate the disordered regions of the molecule
+    """
```

Comment:
Docstring is incomplete.
Also, is it worth making the attribute used an argument (with default "cgidr")?